nym-credential-proxy/nym-credential-proxy/src/storage/mod.rs

```diff
@@ -40,9 +42,15 @@ impl VpnApiStorage {
         let opts = sqlx::sqlite::SqliteConnectOptions::new()
             .filename(database_path)
             .create_if_missing(true)
-            .disable_statement_logging();
+            .log_statements(LevelFilter::Trace)
```

Comment:
Trace by default?

Reply:
i don't think we have to be logging all db statements at higher severity than that : )